tests/conftest.py

```diff
@@ -113,6 +113,14 @@ def pytest_collection_modifyitems(config: Config, items: List[pytest.Item]) -> N
             item.add_marker(pytest.mark.integration)
         elif module_root_dir.startswith("unit"):
             item.add_marker(pytest.mark.unit)
+
+            # We don't want to allow using the script resource if this is a
+            # unit test, as unit tests should not need all that heavy lifting
+            if set(getattr(item, "funcargnames", [])) & {"script"}:
```

Comment:
Is this equivalent to

```python
if "script" in item.funcargnames
```

?

Reply:
Firstly, `funcargnames` was deprecated, now `fixturenames`.
Is it guaranteed that `pytest.Item` has `fixturenames` attribute?